### PR TITLE
New recipe: extra-cmake-modules

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: kde-frameworks/extra-cmake-modules/extra_cmake_modules-5.31.0.recipe [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	extra_cmake_modules = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libQt5Core$secondaryArchSuffix
```
Comment (waddlesplash) [inline kde-frameworks/extra-cmake-modules/extra_cmake_modules-5.31.0.recipe]: Is this really a REQUIRES?

Comment (miqlas) [inline kde-frameworks/extra-cmake-modules/extra_cmake_modules-5.31.0.recipe]: Don't think so.

Comment (waddlesplash) [inline kde-frameworks/extra-cmake-modules/extra_cmake_modules-5.31.0.recipe]: Then you can remove it. :)

**Comment by miqlas** [comment]
Already did it.

**Pull request merged**